### DISCUSSION TITLE
Capture: insert a tab (for a nested list)

### DISCUSSION THREAD

**Discussion by danielhofer** [chhoumann/quickadd#584] [Q&A]
Hi

I'd like to capture ar daily note, but the daily "heading" is not a heading but a list entry in a nested list (with a bullet point).

For a capture I insert the `{{value}}` after a list entry:
`- {{DATE:YYYY.MM.DD}}`
That works fine.

But I'm not able to insert the capture with and indent.
I tried:
`\t- {{value}} `
but it does not work.

Is there a chance to insert a tabstopp in front of the line in the capture format?
I could do it with some spaces at the beginning of the line but it's not a "proper" indent in markdown.

Thanks a lot for a short hint.
Daniel

**Comment by chhoumann** [comment]
Very late answer - sorry, Daniel.

`\t` isn't one of the escapes QuickAdd expands in capture formats (only `\n` and `\\` are), so it lands in the note literally. Two ways to get the indent today:

1. Leading spaces - and good news there: this *is* proper Markdown nesting. Two to four spaces before the `-` is exactly how nested list items are written under a `- parent` line, and Obsidian renders it as a nested list:

```
    - {{VALUE}}
```

with "Insert after" set to your `- {{DATE:YYYY.MM.DD}}` line.

2. If you specifically want a tab character, paste a literal tab into the Capture format field. Typing <kbd>Tab</kbd> moves focus out of the field, but a pasted tab is kept and written to the note as-is.

If you'd like `\t` supported as an escape alongside `\n`, feel free to open a feature request - it's a reasonable ask.
